pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -215,40 +215,50 @@ func (os *OpenStack) AttachDisk(instanceID, volumeID string) (string, error) {
 	if err != nil {
 		return "", err
 	}
+
 	if volume.Status != VolumeAvailableStatus {
```

Comment:
Please add these comment to https://github.com/kubernetes/kubernetes/pull/50042 since this PR is now closed :)